android/Gutenberg/src/main/java/org/wordpress/gutenberg/GutenbergView.kt

```diff
@@ -362,11 +367,7 @@ class GutenbergView : WebView {
 
     @JavascriptInterface
     fun onEditorContentChanged() {
-        getTitleAndContent(object : TitleAndContentCallback {
-            override fun onResult(title: String, content: String) {
-                contentChangeListener?.onContentChanged()
-            }
-        }, false)
```

Comment:
This may have been utilized at one point, but I perceive this `getTitleAndContent` invocation is unnecessary now. The `getTitleAndContent` method is invoked when the host app chooses to persist the latest changes on a set interval (in response to the broadcasted content change event).